server/text_generation_server/cli.py

```diff
@@ -71,7 +72,7 @@ def serve(
     # Downgrade enum into str for easier management later on
     quantize = None if quantize is None else quantize.value
     dtype = None if dtype is None else dtype.value
-    if dtype is not None and quantize is not None:
+    if dtype is not None and quantize is not None and quantize != Quantization.ct2:
```

Comment:
Should I define float16 / bfloat / int8 over dtype -- or -- make them three items in the Quanitzation Enum?